macros/stitch_user_identifiers.sql

```diff
@@ -0,0 +1,13 @@
+{% macro stitch_user_identifiers(enabled, relation=this, user_mapping_relation=ref('snowplow_mobile_user_mapping')) %}  
```

Comment:
Do we plan on applying this macro as a post hook on the sessions table as per the web model?